lib/actions/deploy.coffee

```diff
@@ -58,6 +63,15 @@ performUpload = (image, token, username, url, size, appName) ->
 
 	uploadToPromise(post, size)
 
+uploadLogs = (logs, token, url, buildId, username, appName) ->
+	request = require('request')
+	url = url || process.env.RESINRC_RESIN_URL
```

Comment:
This is the wrong way to get this setting as it ignores the YAML configs. Look for resin-settings-client usages here and in resin-sdk-preconfigured.